TLM/TLM/Manager/Impl/RoutingManager.cs

```diff
@@ -274,7 +274,7 @@ public class RoutingManager
 
                 Services.NetService.IterateNodeSegments(
                     nodeId,
-                    (ushort segId, ref NetSegment segment) => {
+                    (ushort segId, ref NetSegment netSegment) => {
```

Comment:
whats this? unintentional change?

Reply:
ok you fixed name collision.

Reply:
Yeah, it's quite annoying when you open solution, always see 10 errors when they are not really errors, so I fixed what I found.
Funny thing is that parameter is unused, but it must be there because lambda declaration require param with that type.